File: src/main/java/name/hennr/series/stalker/Record/DummyController.java
package name.hennr.series.stalker.Record;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class DummyController {

    @Autowired
    private PersonRepository repository;

    @GetMapping("/record/get")
    public String get() {
        Person person = new Person();
        person.setData("bar");
        repository.save(person);
        return repository.findAll().get(0).getData();
    }
}

File: src/main/java/name/hennr/series/stalker/record/DummyController.kt
package name.hennr.series.stalker.record

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class DummyController {
    @Autowired
    private val repository: PersonRepository? = null
    @GetMapping("/record/get")
    fun get(): String {
        val person = Person()
        person.data = "bar"
        repository!!.save(person)
        return repository.findAll()[0]?.data.toString()
    }
}
